src/dmd/expression.d

```diff
@@ -5876,6 +5926,7 @@ enum MemorySet
 }
 
 /***********************************************************
+ * An expression with `=` or `op=`
```

Comment:
This is not correct. We have `BinAssignExp` for `op=` which is a `BinExp`. You could document that also, while you are at it.